### ISSUE TITLE
Error null is not an object (evaluating 'e.toLowerCase') when switching to Chats or to Wallet [upgrade]

### ISSUE THREAD

**Issue by annadanchenko** [status-im/status-react#3081]
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: Only on release build, does not happen in develop. Error "null is not an object (evaluating 'e.toLowerCase')" is shown after upgrade if sign in and wait for 1-2 minutes.
Found on upgraded app (from 0.9.12 -> release build 65), iOS and Android

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
can switch to Chats and see list of chats
#### Actual behavior
[comment]: # (Describe what actually happened.)
Error null is not an object (evaluating 'e.toLowerCase')

### Dataset to create before upgrade:
- ios and android devices added each other to contacts
- both have default pictures/usernames
- in 1-1 chat: 
ios: text 
android: text
ios: request
android: send (not to request some other amount)
android: text
android: send
android: emoji
android: map
android: request
iOS: send
- Group chat: android, ios and Jarrad
ios: send
android: request
android: map
android: send
android: emoji
ios: enoji
ios: text
ios: map
ios: text

### Genymotion VM with created dataset

You need to have Genymotion
You need to have VirtualBox
Download .ova file https://drive.google.com/open?id=16gW89ua5XLo8201quQhGY-DvbXnjo-LM
In VirtualBox menu select File -> Import. Chose the .ova file you downloaded. And proceed import.
Relaunch Genymotion so it pick up the VM

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)
ios https://app.testfairy.com/projects/4803590-status/builds/7560436/sessions/4/?accessToken=GX5sSxWhocgvQPZp-MVNZOn72hg
android (00:58) https://app.testfairy.com/projects/4803622-status/builds/7525714/sessions/101/?accessToken=9ANlXEl78/niz/TaCBcIy/7-to8
- Open Status
- Sign in and wait for 1-2 minutes

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: release build 65 (upgraded from 0.9.12 to this build)
* Operating System: 
- found on iOS real device iphone 6s and real Android Pixel XL, Android 8.1.0.

**Comment by jeluard** [comment]
Relevant exception

```
00:53 4/Status: 2018-01-18 11:45:56.260 [info][tid:com.facebook.react.JavaScript] DEBUG [status-im.utils.handlers:29] - Handling re-frame event:  :chat-received-message/add
00:53 4/Status: 2018-01-18 11:45:56.297 [warn][tid:com.facebook.react.JavaScript] 'Additional exception info:', 'Error rendering component (in contact-groups-list)'
00:53 4/Status: 2018-01-18 11:45:56.298 [error][tid:com.facebook.react.JavaScript] TypeError: null is not an object (evaluating 'e.toLowerCase')

This error is located at:
    in contact-groups-list
    in reagent82
    in with-activity-indicator
    in reagent25
    in RCTView
    in RCTScrollContentView
    in RCTScrollView
    in ScrollView
    in RCTView
    in e
    in RCTView
    in reagent23
    in RCTView
    in n
    in RCTView
    in t
    in reagent22
    in RCTView
    in main-tabs
    in reagent21
    in RCTView
    in RCTView
    in main
    in root
    in reagent1
    in RCTView
    in RCTView
    in t
```

**Comment by annadanchenko** [comment]
similar bug was mentioned in https://github.com/status-im/status-react/issues/2602#issuecomment-355355979

**Comment by janherich** [comment]
@annadanchenko Can you please test if you are able to trigger the same error on current `develop` build as well ?

**Comment by annadanchenko** [comment]
@janherich I can't reproduce it on last nightly build. tried 4 times with same dataset both ios and android. I've updated issue description with exact dataset I'm using to reproduce the issue

**Comment by asemiankevich** [comment]
I can repro it as well: 

https://app.testfairy.com/projects/4803590-status/builds/7525740/sessions/13/?accessToken=eXQ9P6Mhv742hxTZ39H9fkn38ZE

https://app.testfairy.com/projects/4803622-status/builds/7525714/sessions/37/?accessToken=nmKAq0yCFQQol4gRVNi9JnNQZ8o

![image](https://user-images.githubusercontent.com/28752642/35152865-37aded34-fd35-11e7-9aca-4c7e426dc139.png)

![image](https://user-images.githubusercontent.com/28752642/35152891-46395186-fd35-11e7-8602-6bf6e0efd58f.png)

**Comment by janherich** [comment]
Trying super hard with the same path as described in ticket (all types of chat before upgrade, `/request`, `/send` and `/location` commands in 1-1 chat with and group chat) and I can't really trigger the bug, testing on real device iPhone SE and `0.9.12` from testflight upgraded to latest release build:
Android: http://artifacts.status.im:8081/artifactory/nightlies-local/im.status.ethereum-1d6b94.apk
iOS: https://i.diawi.com/cjSaCP

**Comment by annadanchenko** [comment]
Same issue can be reproduced, see https://github.com/status-im/status-react/pull/3110#issuecomment-359779866

**Comment by annadanchenko** [comment]
should be in release (already there) and in develop

**Comment by yenda** [comment]
@annadanchenko can we close this then ? It is currently in merge column but that's an issue and the related PRs are merged

**Comment by oskarth** [comment]
Closing as it appears to have been fixed in release 0.9.13 / not needed / already done in develop.
